app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,35 @@
+class MoviesController < ApplicationController
+  respond_to :json
+
+  def index
+    user = User.find(params[:user_id])
+    movies = user.movies
+    data = { movies: movies }
+    render json: data, status: 200
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @movie = @user.movies.new(movie_params)
```

Comment:
are you just returning json here? if so, these don't need to be instance variables